p2panda-rs/src/materializer/error.rs

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+use thiserror::Error;
+
+/// Error types for methods of `Materializer` struct.
+#[derive(Error, Debug)]
+#[allow(missing_copy_implementations)]
+pub enum MaterializationError {
+    /// Materialization error.
+    #[error("materialization error")]
+    Error,
```

Comment:
I can't see this error being used anywhere (also it does not describe any particular case), can this be removed?